src/pages/lessons/fundamentals/eth-intro-part-3.mdx

```diff
@@ -0,0 +1,209 @@
+---
+title: A Developer's Guide to Ethereum, Pt. 3
+description: Introduction to Ethereum with web3.py and Python.
+icons: ["ethereum"]
+---
+
+import { LessonHeader } from "../../../components/mdx/LessonHeader";
+import Layout from "../../../components/Layout";
+
+<Layout
+    title="A Developer's Guide to Ethereum, Pt. 3"
+    description="Introduction to Ethereum with web3.py and Python."
+>
+<LessonHeader title="A Developer's Guide to Ethereum, Pt. 3" />
+
+Welcome to Part 3 in the saga! Part 1 covered blockchain fundamentals. Part 2 continued on to examine Ethereum accounts and how they enable participation in the network. Part 3 will build on those concepts with the introduction of smart contracts.
+
+## Smart contracts, an introduction
+
+The Ethereum blockchain has a great deal of value flowing through it. In previous articles, we discussed the flow of **ether** from one user to another via transactions. However, the network is capable of more sophisticated interactions, and those use cases are enabled by smart contracts.
+
+Let's start with a simple definition: a **smart contract** is code (i.e., a computer program) deployed to the blockchain. As for the buzzword name, _contract_ conveys the relative permanence of these programs as they determine how assets can change hands and _smart_ is a nod to their programmability. For brevity, they're commonly referred to as just contracts, and we'll follow suit.
```

Comment:
This is some pretty high level English: `For 'brevity', they're commonly referred to as just contracts, and we'll 'follow suit'.` could you simplify just a part of it?